packages/commerce-sdk-react/scripts/build-and-release-docs.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const sh = require('shelljs')
+const semver = require('semver')
+
+// this script is expected to run on release branch.
+const main = () => {
+    // get the version number from package json
+    const {stdout: currentVersion} = sh.exec(
+        "cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g'"
+    )
+    const isPrerelease = semver.prerelease(currentVersion.trim())
+    // if it is a pre-release version, do nothing
+    if (isPrerelease) {
+        return
+    }
+
+    let latestVersion
+
+    //TODO: Refactor this after we've release first version of commerce-sdk-react
+    // get the latest version from npm registry
+    const {stdout, stderr} = sh.exec('npm view commerce-sdk-react version', {silent: true})
+    latestVersion = stdout
+    // if error, it means 'commerce-sdk-react' is not in the npm registry yet
+    if (stderr) {
+        // use the version of pwa-kit-react-sdk instead
+        const {stdout} = sh.exec('npm view pwa-kit-react-sdk version', {silent: true})
+        latestVersion = stdout
+    }
```

Comment:
This is getting into the weeds, but there would be 2 kinds of errors here. One that we potentially expect, that it's not published yet, and an unknown error like a network issue.

Would it make sense to assuming that the current version is the latest if it returns that it's not in the registry yet? because we are technically running this when we want to release.

If we make that assumption then we don't have to make this second call to check a different npm package. 

If it's an unknown error, then maybe we actually just throw that error and the script fails.